src/stackedchart/StackedChartData.ts

```diff
@@ -0,0 +1,21 @@
+export interface StackedChartData{
+    title: string,
```

Comment:
looks like we still have some charting related field names. I can help change that later.